tmux/formater.go

```diff
@@ -240,13 +241,23 @@ func (f *Formater) divergence() string {
 		return ""
 	}
 
+    var s_behind string
```

Comment:
Please use short form, and underscores are avoided in go for names.
```suggestion
    behind := ""
```

Reply:
Will update! Oh, gotcha!